agent/api/containerstatus.go

```diff
@@ -22,6 +22,13 @@ const (
 	ContainerCreated
 	// ContainerRunning represents a container that has started
 	ContainerRunning
+	// ContainerResourcesProvisioned represents a container that has completed provisioning all of its
+	// resources. Non-internal containers (containers present in the task definition) transition to
```

Comment:
I really like this idea, in this way we will get rid of the error for starting the `volumes` container, and save a `docker start` API call for the `volumes` container.